### PR TITLE
[DEV-16314] add support for amqp resources

### PR DESCRIPTION
**Pull request by valeri-sokolov-getpfc**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -12,6 +12,10 @@ ignored = [
   name = "github.com/gorilla/websocket"
   version = "v1.2.0"
 
+[[constraint]]
```
Comment (7phs) [inline Gopkg.toml:15]: :-( dep

File: resource_test.go [hunk 2]
```diff
@@ -43,6 +43,9 @@ func TestParseResourcesSuccess(t *testing.T) {
 
 		"mysql://user:pass@localhost:3306/dbname?query=val#fragment",
 
+		"amqp://host:5672/vhost",
```
Comment (7phs) [inline resource_test.go:46]: Does it make sense to add an alias "rabbitmq" for it?

Comment (valeri-sokolov-getpfc) [reply resource_test.go:46]: I'd rather not invent non-standard names here

**Review by 7phs — APPROVED** [review]

**Review by HaraldNordgren — APPROVED** [review]

**Comment by djui** [comment]
@valeri-sokolov-getpfc please also update the README.md .

**Comment by fcristovao** [comment]
I thought this repository was no longer maintained? I'm happy to see it come back to life :D 
But now it will have diverged, as I had already created a fork that included kafka resources.
Any ideas?

**Comment by HaraldNordgren** [comment]
Hi, Felipe!

We are maintaining the repo based on the needs that we have within the company. But we don't have the capacity right now to handle external feature development. Hope that makes sense 😃

**Comment by fcristovao** [comment]
Aha, ok. Thank you!

**Pull request merged**